views/default/input/access.php

```diff
@@ -66,7 +66,7 @@
 }
 
 // don't call get_default_access() unless we need it
-if (!isset($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
+if (!isset($vars['value']) || is_null($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
```

Comment:
I get a missing label option. Value is set but its === null
On Apr 17, 2015 2:18 PM, "Steve Clay" <notifications@github.com> wrote:

> In views/default/input/access.php
> <https://github.com/Elgg/Elgg/pull/8216#discussion_r28588728>:
>
> > @@ -66,7 +66,7 @@
> >  }
> >
> >  // don't call get_default_access() unless we need it
> > -if (!isset($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
> > +if (!isset($vars['value']) || is_null($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
>
> Actually I think this PR may not be needed. !isset implies missing key or
> value is null.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8216/files#r28588728>.
>